README.md

```diff
@@ -26,12 +26,18 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
 default['supermarket_omnibus']['package_url'] = 'http://bit.ly/98K8eH'
 ```
 
-If you wish to use custom SSL certificates, define the following `config` attributes:
+If you wish to specify additional settings, you can pass them via the `default['supermarket_omnibus']['config']` attribute.  
+Example: for custom SSL certificates define the following `config` attributes:
 
 ```ruby
 default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/ssl.crt'
 default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/ssl.key'
 ```
+Above attributes, if defined in supermarket.rb directly, would look like this:
```

Comment:
I was referencing this article. https://docs.chef.io/config_rb_supermarket.html
Out of the box, supermarket.rb has all of these settings defined and commented out. ( I like this approach a lot btw )

Initially I had supermarket.json for half of the attributes (due to bootstrap method of setting up supermarket) and the other half at the supermarket.rb (due to manually setting items). And everything worked splendidly (as expected).

I updated the wrapper for this cookbook to move attribs defined in supermarket.rb into the wrapper, remove them from supermarket.rb, and again, everything worked fine, which is what I expected.

@irvingpop - I don't have a particular point, other than I want ['config'] hash documentation written out in a way that's very clear to understand :)